release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -135,6 +135,11 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster via an alert on the *Overview* -> *Cluster* page, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement(SLA)* of the *Overview* page.
```

Comment:
`an alert on the *Overview* -> *Cluster* page` , we show SLA info in Home -> Overview -> Details card
`in the *Cluster Settings*`, shall we use `in Cluste Settings page` or `on Cluster Settings page` ? 
attaching a screenshot for reference
<img width="565" alt="Screen Shot 2022-02-17 at 3 31 57 PM" src="https://user-images.githubusercontent.com/12692381/154427124-b14878e6-171a-4fb2-9b3e-607f069848f5.png">

Reply:
`You can manage subscription settings under the *Service Level Agreement(SLA)* of the *Overview* page`  Shall we update it to something like `You can manage subscription settings by clicking `Manage subscription settings` link under the *Service Level Agreement(SLA)* of the *Overview* page`

Reply:
also we have SLA info can not be shown correctly due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2048059